.github/workflows/ci.yml

```diff
@@ -22,5 +22,23 @@ jobs:
         npm install
         npm run build:v2-compat
         npm run build:firefox
+        mv dist/ firefox/
```

Comment:
After this line (and with subsequent repetitions) there should be a `npm run clean` to cleanup the residual files from the previous build

Reply:
@Bill13579 please consider committing WIP and will work on followup PRs

GHA wants a configuration on the default branch